src/services/File Management/rmq/pushToRMQ.js

```diff
@@ -0,0 +1,36 @@
+const amqp = require("amqplib");
+const prisma = require("../prisma/prismaClient");
+const RMQURL = process.env.RMQURL;
```

Comment:
This line will fail because this file is called before this line `require("dotenv").config();` in `server.js`.